support-frontend/assets/pages/digital-subscriber-checkout/components/billingPeriodSelector.tsx

```diff
@@ -59,6 +59,17 @@ const subheading = css`
 	${textSans.medium()};
 `;
 
+const choiceCardContainer = css`
+	div {
+		${from.mobileLandscape} {
+			column-gap: ${space[1]}px;
+		}
+		${from.tablet} {
+			column-gap: ${space[2]}px;
+		}
+	}
+`;
```

Comment:
It does feel as though this should be just part of what's used on the `/contribute` checkout, but probably worth drawing in where we try to abstract it.